controllers/v1/userProjects.js

```diff
@@ -1125,5 +1126,83 @@ module.exports = class UserProjects extends Abstract {
                 }
         })
     }
+    /**
+    * @api {post} /improvement-project/api/v1/userProjects/listPendingProjects
+    * Lists projects based on the status 
+    * @apiVersion 1.0.0
+    * @apiGroup User Projects
+    * @apiSampleRequest /improvement-project/api/v1/userProjects/listPendingProjects
+    * @apiParamExample {json} Response:
+    /**
+
+    /**
+     * Lists projects based on the status
+     * @method
+     * @name listPendingProjects
+     * @returns {JSON} Project details
+    */
+
+    async listPendingProjects(req) {
```

Comment:
we can have only one controller @borkarsaish65

Reply:
controller can named as usersProject

Reply:
I could do that but I used the same code which I had created for previous task given by you. So there we were asked list projects based on the type passed so its the same.

The second point of creating two different controller is that one is queried by userId field and another is queried by createdBy field.

Let me know what you think ?

Reply:
no that not its should work and now we have only two task 
big number and project details that can be done in one API 
create only one controller @borkarsaish65

Reply:
Done